### ISSUE TITLE
Some items fail the color contrast

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/rutherford-scattering#184]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Chrome

**Problem description**
From https://github.com/phetsims/qa/issues/1353

Some items fail the color contrast with the projector mode option on:

- In the Rutherford Atom screen: white background (#FFFFFF) Vs Traces (#EB9DF9) --> 1.95:1

<img width="44" height="28" alt="Image" src="https://github.com/user-attachments/assets/7ee95bf2-493d-4caf-963c-c76a3541555d" />

- In the Plum Pudding Atom screen: positive charge (#C14000) Vs Traces (#808080) --> 1.32:1 

<img width="35" height="24" alt="Image" src="https://github.com/user-attachments/assets/0328092a-ba41-464c-8aed-7acaeb16dde9" />

- In the Plum Pudding Atom screen: electrons (#6D71B4) Vs positive charge (#C14000) --> 1.16:1

<img width="32" height="22" alt="Image" src="https://github.com/user-attachments/assets/3e242699-cd0d-4472-a306-03701093e22f" />

- In the Plum Pudding Atom screen: Green edge of the Alpha Particles Button (#058394) Vs Brown part of the Alpha Particles button (#C49B73) --> 1.76:1 

<img width="54" height="63" alt="Image" src="https://github.com/user-attachments/assets/0fbf6f66-97f1-4bed-b68b-73c5e71cccbe" />

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Rutherford Scattering‬
URL: https://phet-dev.colorado.edu/html/rutherford-scattering/1.2.0-dev.3/phet/rutherford-scattering_all_phet.html
Version: 1.2.0-dev.3 2026-01-30 16:13:37 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/144.0.0.0 Safari/537.36
Language: en-US
Window: 1440x731
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by brent-phet** [comment]
I'll try to get you and answer by tomorrow Friday 2/6. @AgustinVallejo 

> [@kathy-phet](https://github.com/kathy-phet) [@brent-phet](https://github.com/brent-phet) Yes/no question: Is it worth like at most 1h of designer time to reconsider the above colors for Rutherford? Or should I defer this for this republication?

**Comment by marlitas** [comment]
> can we add a black stroke around the electrons?

Unfortunately the electrons in the plum pudding atom screen are part of a png and not drawn on... If I get just the plum pudding background as an asset (the icon png is too small) I can draw the electrons myself and have more control on that.

Everything else was done in the below commit.

**Comment by marlitas** [comment]
Okay this is ready for review again @arouinfar. Let me know if there's anything that needs to be done here, if not feel free to close.

**Comment by arouinfar** [comment]
Thanks @marlitas the color changes all look good. The electrons, however, look pretty pixelated (even when zoomed out in the smaller screenshot) compared to the particles in the legend. As a result the stroke around the electrons looks muddy and doesn't improve the contrast as I'd hoped. 

<img width="110" height="75" alt="Image" src="https://github.com/user-attachments/assets/cb9fe5f4-4078-4e1d-b8be-fbc49af7424b" />

<img width="285" height="142" alt="Image" src="https://github.com/user-attachments/assets/5b34dd1d-f44e-44e8-9c8d-cffe96f7282a" />

<img width="305" height="534" alt="Image" src="https://github.com/user-attachments/assets/055f4b9d-30be-4089-b3c7-be2be31a0d7c" />

**Comment by arouinfar** [comment]
Much better now, thanks @marlitas! Closing.
